commander/src/utils/reader.ts

```diff
@@ -221,6 +221,7 @@ const castValue = (
 		return JSON.parse(val);
 	}
 	if (schemaType === 'array') {
+		if (val === '') return [];
```

Comment:
```suggestion
		if (val === '') {
			return [];
		}
```

Reply:
Pushed a new implementation which uses the ternary operator.